index.js

```diff
@@ -551,6 +552,19 @@ Browserify.prototype._createDeps = function (opts) {
                 return through();
             }
         }
+
+        if (opts.commondir === false && opts.builtins === false) {
+          opts.insertGlobalVars = xtend({
```

Comment:
could this be done in cmd/args.js, in the same place as the other `node:` option stuff?

Reply:
Not sure I understand this comment. Do you mean setting `commondir: false` and `builtins: false` if `node: true`, and then just check for `node: true` here?

Reply:
oh sorry nevermind, i misunderstood what this check is doing--i thought you were using this to essentially check if `--node` was passed, but `opts.commondir` actually is the one causing the issue.